app/views/posts/index.html.erb

```diff
@@ -1 +1,26 @@
-<h1>Lists of post for a given user</h1>
\ No newline at end of file
+<section class="container">
+  <div class="user-header"> 
+    <div class="user-icon">
+      <i class="fa fa-user"></i>
+    </div>
+    <div class="user_name">
+      <h1><%=@user.name%>
+    </div>
+  </div>
+  <% @posts.each_with_index do |post, index| %>
+      <%=link_to user_post_path(@user.id,post.id) do %>
+        <div class="user_post">
+            <div>
+              <h3>Post #<%=index+1%>: <%= post.title %></h3>
+              <p><%= post.text %></p>
+            </div>
+              <h4><span>Comment <%= post.commentscounter %></span> <span>|</span> <span>Likes <%= post.likescounter %></span></h4>
+        </div>
+      <% end %>
+      <% @commets.each do |comment|%>
+      <div>
+        <p><%=comment.Text%></p>
+      </div>
+      <%end%>
```

Comment:
All comments should be displayed on the show page. You may use the recent_comments methods created in previous activities to display recent comments on index page
### For Example
```ruby
<% post.recent_comments.each do |comment| %>
  <p>
    <%= comment.author.name %>: <%= comment.text %>
  </p> 
<% end %>
```

Reply:
Thanks for your review let me work on the changes